pymc3/sampling.py

```diff
@@ -427,8 +427,10 @@ def sample(
         check_start_vals(model.test_point, model)
     else:
         if isinstance(start, dict):
+            start = {k: v for k, v in start.items()}
```

Comment:
Why not simply `start.copy()`?

Reply:
This is what I was suggesting initially,. Anyway, I will replace that with `deepcopy`.

Reply:
This line can be removed now.